_config.yml

```diff
@@ -95,6 +95,7 @@ JB :
       colorscheme: light
    
   # Settings for analytics helper
+  # Set 'site.safe' to true to enable analytics plugin
```

Comment:
@themez, double crap. This now has merge conflicts. Will have to create new PR. If you don't have time or desire, I'll handle it. Thanks for contribution!